Grid/Grid.php

```diff
@@ -632,7 +633,7 @@ protected function processMassActions($actionId)
                         $action->getParameters()
                     );
 
-                    $subRequest = $this->container->get('request')->duplicate([], null, $path);
+                    $subRequest = $this->request->duplicate(array(), null, $path);
```

Comment:
This sounds good but we have a kind of mix in the code: sometimes we use `$this->{sometingRetrievedFromContainer}` and sometimes we use `$this->container->get({serviceName})` like line 638.

Don't know if is worth to keep same logic and initialize private members at construct or not. 

Any opinions?